pkg/config/lima_config_applier.go

```diff
@@ -188,3 +235,74 @@ func hasUserModeEmulationInstallationScript(limaCfg *limayaml.LimaYAML) (int, bo
 
 	return scriptIdx, hasCrossArchToolInstallationScript
 }
+
+// initializes the bool slice for what snapshotter the user has enabled to all false
+// this will be changed later depending on the user's snapshotters config values.
+func initializeEnabledSnapshotterSlice(numSupportedSnapshotters int) [2][2]bool {
+	var enabledSnapshotters [2][2]bool
```

Comment:
I don't think we need this function. Go will set uninitialized values to the type's "zero" value, which for a `bool` is `false`. e.g.

```
var b [2][2]bool
fmt.Printf("%+v\n", b)

[[false false] [false false]]
```